backend/hdf5/DimensionHDF5.cpp

```diff
@@ -259,13 +269,153 @@ void SetDimensionHDF5::labels(const none_t t) {
 
 SetDimensionHDF5::~SetDimensionHDF5() {}
 
+//--------------------------------------------------------------
+// Implementation of DataFrameDimensionHDF5
+//--------------------------------------------------------------
+DataFrameDimensionHDF5::DataFrameDimensionHDF5(const H5Group &group, ndsize_t index)
+    : DimensionHDF5(group, index)
+{
+    setType();
+    // Note: maybe having this is a bad idea, because a DataFrameDimension created
+    // this way will not have the block and file
```

Comment:
Do we need it, if not remove it?

Reply:
maybe help me answering the question. RangeDimension and also the DataFrameDimension rely on file and block, but all Dimensions have this kind of constructor. 
I would rather remove the constructors.

Reply:
we can not remove the constructors, removed the note